gridData/OpenDX.py

```diff
@@ -885,8 +888,20 @@ def __array(self):
                             'array: Only the "data follows header" format is supported.')
             if not self.currentobject['size']:
                 raise DXParseError("array: missing number of items")
-            self.currentobject['array'] = [self.__consume().value('REAL') \
-                                           for i in range(self.currentobject['size'])]
+            # This is the slow part.  Once we get here, we are just
+            # reading in a long list of numbers.  Conversion to floats
+            # will be done later when the numpy array is created.
+
+            # Don't assume anything about whitespace or the number of elements per row
```

Comment:
Yes – keep it more general. Thanks for the comments in the code.

Reply:
Sorry, do you want me to delete the alternate version or leave it as is?

Reply:
Leave it.